<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionMaintenancePage</class>
 <widget class="QWidget" name="CollectionMaintenancePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>371</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="verticalLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Folder type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="folderTypeLbl">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="folderSizeLbl">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>File size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Total items:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="itemsCountLbl">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Unread items:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="unreadItemsCountLbl">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="indexingLabel">
     <property name="text">
      <string>Indexing:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="enableIndexingChkBox">
     <property name="text">
      <string>Enable fulltext indexing</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="indexedCountLbl">
     <property name="text">
      <string>Retrieving indexed items count...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="reindexButton">
     <property name="text">
      <string>Reindex folder</string>
     </property>
    </widget>
   </item>
   <item row="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
